client/types.go

```diff
@@ -635,3 +635,17 @@ type PolicyConfig struct {
 type PoliciesConfigList struct {
 	Policies []PolicyConfig `json:"policies_config"`
 }
+
+// OIDCConfigurationItem - Holds an OIDC configuration item object
+type OIDCConfigurationItem struct {
+	ID                        int64 `json:"id,omitempty"`
```

Comment:
Is there some reason ID has omitempty struct tag and the others don't?

Reply:
When reading OIDCConfiguration from 3scale API, the ID will be set. 
When updating OIDCConfiguration of your product, the serialized object is that one and `ID` field is not required to be sent. It not `omitempty`, the request would  send a `ID: 0` field which is unwanted.

Reply:
I see. :+1: